src/gradlecheck/CreateMarkDownTable.groovy

```diff
@@ -21,7 +21,7 @@ class CreateMarkDownTable {
     }
 
     def createMarkdownTable() {
-        def groupRows = this.tableData.groupBy{it.gitReference }
+        def groupRows = this.tableData.findAll { it.gitReference && !["2.19", "2.x", "1.x"].contains(it.gitReference)}.groupBy{ it.gitReference }
```

Comment:
Instead can we have just look up for `main` and ignore rest, this is to avoid adding multiple versions or .x branches in future.

Reply:
I mean it should be possible right when you gitReference what it contains and move forward if it contains main.

Reply:
I got it now. So we create everything for main and commitsID's. So this code works for all except "2.19", "2.x", "1.x" ?

Reply:
Yes Prudhvi. I will filter only main and commitid's on the issue body